base/task/post_job_unittest.cc

```diff
@@ -45,6 +46,9 @@ TEST(PostJobTest, CreateJobSimple) {
       CreateJob(FROM_HERE, {}, BindLambdaForTesting([&](JobDelegate* delegate) {
                   EXPECT_TRUE(job_started);
                   barrier.Run();
+#if defined(USE_HACKY_COBALT_CHANGES)
```

Comment:
So this macro is only used temporarily to enable compilation of upstream libraries, if this is for Starboard, shall we use #if defined(Starboard)?